ui/packages/catalog-ui-search/src/main/webapp/component/layer-item/layer-item.hbs

```diff
@@ -22,11 +22,19 @@
 </button>
 <div class="layer-properties">
     <div class="layer-name" title="{{name}}">{{name}}</div>
-    <div class="layer-visibility group">
-        <button class="is-button layer-show">
-            <span class="fa fa-eye"></span>
-            <span class="fa fa-eye-slash"></span>
-        </button><!--
+    <div class="layer-visibility"><!--
     --><input class="layer-alpha" value={{alpha}} min="0" max="1" step="0.01" type="range"/>
     </div>
-</div>
\ No newline at end of file
+    <div class="layer-interactions">
+        <button class="is-button layer-warning" data-help="View map layer warnings.">
+            <span class=" fa fa-warning" title="{{warning}}"></span>
```

Comment:
✏️ Move the title to the button, otherwise it'll only show a tooltip if you hover right on the icon.